ios/emurgo/Images.xcassets/AppIcon-nightly.appiconset/Contents.json

```diff
@@ -1,62 +1,62 @@
 {
-  "images" : [
```

Comment:
I'd would remove everything under `ios/` too beacuse it may conflict with Xcode's internal linting, creating unnecessary code diffs in the future.

Reply:
done